### PR TITLE
Add a default vconsole.conf to the boot.iso

### PR DESCRIPTION
**Pull request by bcl**

### CODE REVIEW

File: share/config_files/common/vconsole.conf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+KEYMAP=us
+FONT=eurlatgr
```
Comment (vpodzime) [inline share/config_files/common/vconsole.conf]: Why this font? I remember using ``latarcyrheb`` in anaconda by default.

Comment (M4rtinK) [inline share/config_files/common/vconsole.conf]: Also with https://github.com/rhinstaller/anaconda/commit/515f0679276369713eaed7d33f86904f9fb062db there should always be a correct font set for the currently selected language. So isn't this change redundant ?

**Comment by bcl** [comment]
This is for the default console font at boot time, and is related to https://fedoraproject.org//wiki/Changes/NewDefaultConsoleFont

Which isn't working right in f23/rawhide. Run 'setfont eurlatgr' and you'll see the font shift. adamw has tested this and confirmed that it fixes the problem.

**Comment by vpodzime** [comment]
Thanks for the explanation! Looks good to me then.

**Comment by bcl** [comment]
pushed.

**Pull request closed without merging**